src/app/flight-booking/components/flight-booking-info-box/flight-booking-info-box.component.ts

```diff
@@ -11,8 +12,37 @@ export class FlightBookingInfoBoxComponent implements OnInit {
   constructor() { }
 
   @Input() departingFlight: Flight;
+  @Input() returnFlight: Flight;
+  @Input() tripType: TripType;
+  @Output() bookFlightButtonClick: Flight;
```

Comment:
why is this an output? shouldn't an output be an event emitter?

Reply:
Yes I didn't hookup the button yet. And missed it.